src/org/usfirst/frc/team2706/robot/subsystems/Bling.java

```diff
@@ -0,0 +1,153 @@
+package org.usfirst.frc.team2706.robot.subsystems;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.usfirst.frc.team2706.robot.controls.StickRumble;
+
+import edu.wpi.first.wpilibj.SerialPort;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Bling extends Subsystem {
+    
+    SerialPort blingPort = new SerialPort(9600, SerialPort.Port.kMXP);
+    int pixels = 120; //The number of pixels on one LED strip
+    boolean batCritical = false; 
+    /*Will be true if the battery level is critical, in which case it will override all
+    *other signals to display the critical battery warning
+    */
+
+    //Let's make the colour and command codes
+    Map<String, String> colours = new HashMap<String, String>(){{
+    put("RED", "16711680");
+    put("GREEN", "32768");
+    put("YELLOW", "65535");
+    put("PURPLE", "8388736");
+    put("ORANGE", "16753920");
+    put("BLUE", "255");
+    put("VIOLET", "15631086");
+    put("MERGE", "5374156");
+    put("TAN", "16767411");
+    put("PINK", "14027935");
+    put("WHITE", "16777215");
+    put("TURQUOISE", "65535");
+    put("BLACK", "0");
+    }};
+    
+	public Bling() {
+	    
+	    blingPort.setTimeout(0.5); //Will wait a max of half a second.
+	    blingPort.writeString("I");
+		String readString = blingPort.readString(); //Get output from the arduino.
+		
+	}
+	
+	/**
+	 * This function should be run at the beginning of autonomous to get
+	 * the proper light pattern.
+	 */
+	public void auto() { //Will run during autonomous period
+	    
+		blingPort.writeString("I");
+		blingPort.writeString("E0Z"); //Clear the LED strip
+		blingPort.writeString("E1Z");
+	}
+	
+	/**
+	 * This function runs to initialize the teleop patterns.
+	 * Starts the teleop period for the bling
+	 */
+	public void startTelOp() {
```

Comment:
Its good I just thing it should be startTeleop and not startTelOp, just looks a little better